sphinx_github_changelog/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from sphinx_github_changelog import changelog
+from sphinx_github_changelog import metadata as _metadata_module
+
+__all__: list = []
+
+
+_metadata = _metadata_module.extract_metadata()
+__author__ = _metadata["author"]
+__author_email__ = _metadata["email"]
+__license__ = _metadata["license"]
+__url__ = _metadata["url"]
+__version__ = _metadata["version"]
+
+
+def setup(app):
+    token_name = "sphinx_github_changelog_token"
+    app.add_config_value(
+        name=token_name, default=os.environ.get(token_name.upper()), rebuild="html"
+    )
+    app.add_directive("changelog", changelog.ChangelogDirective)
+
+    return {
+        "version": "0.1",
```

Comment:
```suggestion
        "version": __version__,
```

Reply:
Indeed. This wasn’t used before but now that’s it’s a full fledged extension, we can set this properly